homeassistant/components/caldav/calendar.py

```diff
@@ -259,6 +262,8 @@ def update(self):
             "summary": vevent.summary.value,
             "start": self.get_hass_date(vevent.dtstart.value),
             "end": self.get_hass_date(self.get_end_date(vevent)),
+            "created": convert_date(vevent.dtcreated.value),
+            "updated": convert_date(vevent.dtupdated.value),
```

Comment:
Replacing convert_date with self.get_hass_date results in 'str' object has no attribute 'isoformat'